frontend/src/components/eMIB/BackgroundSection.jsx

```diff
@@ -0,0 +1,129 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import ReactMarkdown from "react-markdown";
+import LOCALIZE from "../../text_resources";
+import ImageZoom from "react-medium-image-zoom";
+import PopupBox, { BUTTON_TYPE } from "../commons/PopupBox";
+import "../../css/react-medium-image-zoom.css";
+import TreeNode from "../commons/TreeNode";
+import { processTreeContent } from "../../helpers/transformations";
+import emib_sample_test_org_chart_en from "../../images/orgCharts/emibSampleTest/org_chart_en.png";
+import emib_sample_test_org_chart_fr from "../../images/orgCharts/emibSampleTest/org_chart_fr.png";
+import emib_sample_test_team_chart_en from "../../images/orgCharts/emibSampleTest/team_chart_en.png";
+import emib_sample_test_team_chart_fr from "../../images/orgCharts/emibSampleTest/team_chart_fr.png";
+
+const styles = {
+  testImage: {
+    width: "100%",
+    maxWidth: 600
+  },
+  button: {
+    marginLeft: 5
+  }
+};
+
+class BackgroundSection extends Component {
+  static propTypes = {
+    content: PropTypes.array,
+    currentLanguage: PropTypes.string
+  };
+
+  state = {
+    showPopupBox: false
+  };
+
+  openPopup = () => {
+    this.setState({ showPopupBox: true });
+  };
+
+  closePopup = () => {
+    this.setState({ showPopupBox: false });
+  };
+
+  render() {
+    const { content, currentLanguage } = this.props;
+    return (
+      <div>
+        {content.map((contentItem, id) => {
+          if (contentItem.type === "markdown") {
+            // Markdown sections are rendered using ReactMarkdown.
+            return <ReactMarkdown key={id} source={contentItem.text} />;
+          } else if (contentItem.type === "tree_view") {
+            // Process the tree content.
+            let treeType = contentItem.organizational_structure_tree_child
+              ? "organizational_structure_tree_child"
+              : "team_information_tree_child";
```

Comment:
At some point we need to just merge these two types.

Reply:
I agree, there is no difference between them!